### PR TITLE
Pull request for Hamayoun portfolio desktop version

### PR DESCRIPTION
**Pull request by Hamayounali**
1. Updated style.css file
2. Updated index.html file
3. Added images

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -1,360 +1,869 @@
...
+  }
+
+  .flex-menu-bar > div {
+    font-size: 30px;
+    color: white;
+    flex-grow: 1;
+  }
+
+  .flex-menu-bar2 {
+    display: flex;
```
Comment (ahmetbozaci) [inline assets/css/style.css:89]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: index.html [hunk 2]
```diff
@@ -65,12 +80,37 @@
 	</div>
 </section>
 
+<div class="hide">
+	<nav class="flex-menu-bar2">
+	<div class="brand">
+		Mamayoun Ali
+	</div>
```
Comment (ahmetbozaci) [inline index.html:87]: - [ ] Brand position is broken in the mobile version please fix it.

image

- [ ] It has some space left and on top of the page please fix it.

image

File: index.html [hunk 3]
```diff
@@ -80,35 +120,50 @@
...
+				<h1 class="work-sample-2"> Multy-Post Stories </h1>
+				<div class="work-sample-detail">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+				</div>
+				<ul class="tags-2">
+					<li class="tag">css</li>
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+				</ul>
+				<a class="see-2" href="#"> See Project</a>
```
Comment (ahmetbozaci) [inline index.html:161]: - [ ] This project section layout is different than the original design. Please fix it. (mobile version)
image

- [ ] See project button's text should center. 

image

- [ ] Also please apply all buttons hover, active disable style. Some of them have not.


image

File: index.html [hunk 4]
```diff
@@ -182,76 +237,151 @@ <h1 class="work-sample"> Profesional Art Printing Data </h1>
...
+	</div>
+	</div>
 </section>
 <br> <br> <br>
 <section class="head">
 	<!-- Heading Section  -->
 	<div class="flex-about">
-	  <div class="about-me">
+	<div class="row">
+	  <div class="about-me col-7">
```
Comment (ahmetbozaci) [inline index.html:316]: - [ ] It has some white space left in the section please fix it.

image

- [ ] Please align the center of the get my resume button's text. Also, apply hover, disable and active style on it.
- [ ] This section layout is broken in the mobile view. Please fix it.
image
___
Original
image

File: index.html [hunk 5]
```diff
@@ -182,76 +237,151 @@ <h1 class="work-sample"> Profesional Art Printing Data </h1>
...
+			<ul class="lang">
+				<li class="lang-title"> <img src="assets/images/rectangle.png" alt="rec">Skills</li>
+				<li>CodeKit</li>
+				<li>GitHub</li>
+				<li>Code Pen</li>
+				<li>GitLab</li>
+				<li>Terminal</li>
+			</ul>
+		</div>
+
```
Comment (ahmetbozaci) [inline index.html:359]: - [ ] This section layout(in the mobile view) is different than original design please fix it.

image
___
Original
image

File: index.html [hunk 6]
```diff
@@ -182,76 +237,151 @@ <h1 class="work-sample"> Profesional Art Printing Data </h1>
...
-		<div>
+	<div class="row">
+		<div class="col-5">
 			<p class="footer-datail">
 				I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
 		 	</p>
-		</div>
-		<div>
+    	</div>
+		<div class="col-7">
```
Comment (ahmetbozaci) [inline index.html:371]: - [ ] Form section design(mobile view) different than original design please fix it.

image
___
Original
image

File: index.html [hunk 7]
```diff
@@ -182,76 +237,151 @@ <h1 class="work-sample"> Profesional Art Printing Data </h1>
...
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+		</div>
+	</div>
+	</div>
```
Comment (ahmetbozaci) [inline index.html:309]: - [ ] I cant see any `see a project button` on these projects. Please add it

image
___
Original
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @Hamayounali  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Well done with adding desktop view. You should make sure the mobile view stays" the same as the original design. 

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: index.html (hunk 7 above)
Comment (Hamayounali) [reply index.html:309]: Thank you for the corrections. Please check it by Inspect -> mobile devices -> Responsive

File: index.html (hunk 6 above)
Comment (Hamayounali) [reply index.html:371]: Thank you for the corrections, I applied that. Please check it by Inspect -> mobile devices -> Responsive

File: index.html (hunk 4 above)
Comment (Hamayounali) [reply index.html:316]: Thank you for the corrections, I applied that. Please check it by Inspect -> mobile devices -> Responsive

**Review by descholar-ceo — APPROVED** [review]
Hi @Hamayounali , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:+1: Implemented all requirements :heavy_check_mark: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Hamayounali** [comment]
Thank you all, for your nice support

**Comment by Hamayounali** [comment]
Thank you all, for your nice support

**Pull request merged**